packages/runtime/container-runtime/src/garbageCollectionHelpers.ts

```diff
@@ -35,3 +38,19 @@ export function sendGCUnexpectedUsageEvent(
 
 	mc.logger.sendTelemetryEvent(event, error);
 }
+
+/**
+ * In order to protect old documents that were created at a time when known bugs exist that violate GC's invariants
+ * such that enforcing GC (Fail on Tombstone load/usage, GC Sweep) would cause legitimate data loss,
+ * the container author may increment the feature support value for Tombstone such that containers created
+ * with a different value will not be subjected to GC enforcement.
+ * @param persistedValue - The persisted feature support value
+ * @param currentValue - The current app-provided feature support value
+ * @returns true if GC Enforcement (Fail on Tombstone load/usage, GC Sweep) should be disabled
+ */
+export function shouldDisableGcEnforcement(
```

Comment:
I am wondering if we should use enable instead of disable to indicate if a feature is enabled or not. I guess it depends on what we expect the primary use of GcFeatureSupportDimensions is - a means to enable features or disable them.

Reply:
Yeah I got the same feedback from Tyler.  I'll do it in a quick follow-up PR

Reply:
Did it now.